mmcv/cnn/utils/flops_counter.py

```diff
@@ -116,7 +117,9 @@ def get_model_complexity_info(model,
     return flops_count, params_count
 
 
-def flops_to_string(flops, units='GFLOPs', precision=2):
+def flops_to_string(flops: float,
+                    units: str = 'GFLOPs',
```

Comment:
```suggestion
                    units: Optional[str] = 'GFLOPs',
```
The value of units can be None so its type annotation should be `Optional[str`.